src/modules/feedback/components/FeedbackBoxContainer.js

```diff
@@ -66,16 +69,16 @@ export class FeedbackBoxContainer extends React.Component<PropsType, StateType>
   }
 
   postFeedbackData = async (feedbackData: FeedbackParamsType) => {
-    const { postFeedbackDataOverride } = this.props
-
+    const { postFeedbackDataOverride, onSubmit } = this.props
     try {
```

Comment:
Move the try/catch down below in `handleSubmit` to ensure everything is really catched and handled.

Reply:
The error should also be logged nevertheless.